py/picca/delta_extraction/expected_fluxes/true_continuum.py

```diff
@@ -95,7 +95,7 @@ def __init__(self, config):
 
         # read large scale structure variance
         self.get_var_lss = None
-        self.read_var_lss()
+        self.read_raw_statistics()
```

Comment:
Before this line I'm missing a `self.get_mean_flux = None`. All members should be declared in the __init__ function

Reply:
@cramirezpe - Don't you need to declare get_var_lss as well? I believe you need both, isn't it?

Reply:
Yes, it has to be there as well.